lib/condor.py

```diff
@@ -153,7 +153,7 @@ def submit(
     #    if True:
 
     qargs = " ".join([f"'{x}'" for x in cmd_args])
-    cmd = f"/usr/bin/condor_submit -pool {COLLECTOR_HOST} {schedd_args} {qargs}"
+    cmd = f"/usr/bin/condor_submit -unused -pool {COLLECTOR_HOST} {schedd_args} {qargs}"
```

Comment:
Well, this goes along with the next two.   See:
https://htcondor.readthedocs.io/en/latest/users-manual/submitting-a-job.html?highlight=conditional%20#using-conditionals-in-the-submit-description-file

It turns out when the conditional isn't true, condor warns about unused lines in the submit file. 
Hence I wanted to squelch that warning, as I figured it would confuse users. 

However you make an excellent point also, we actually know at generation time whether that
conditional is going to be true (we don't have --dag or --dataset-dag) so I could not have a
submit-time conditional at all, and then I wouldn't need the flag.

Reply:
Okay, I pushed a change to clean these up.